website/docs/user-guides/reset-password.md

```diff
@@ -20,6 +20,13 @@ In the **Reset Password** modal type in the new password in the **New Password**
 
 <img src={require('../assets/user-guides/user-management/reset-password/step-2.png').default} />
 
+
+## 3. Update password after installation
+
+After installing Chaos Litmus and entering the default passwords, users will be redirected to the update password page, which is mandatory. In the update password modal, enter your Old Password, New Password, and Re-enter New Password. Once done, click the Confirm button.
```

Comment:
```suggestion
After installing LitmusChaos and authenticating using the default password in the UI, users will be mandatorily prompted to update the password. To update the password, enter the old (default) password, the new password, and re-enter the new password. Once done, select the **Confirm** button.
```

Reply:
Thank you for the suggestion, I have made the changes accordingly.